wlct/tournaments.py

```diff
@@ -1815,6 +1819,9 @@ def start(self, tournament_data):
                     self.create_game(tournament_round, game)
         except Exception:
             log_exception()
+        finally:
+            self.process_new_games_allowed = True
```

Comment:
Is it possible for there to be an exception where the tourney wouldn't be started? Might need to move this in the try block